rise/static/main.js

```diff
@@ -615,25 +671,36 @@ function revealMode() {
   }
 }
 
+// users can bind another key with e.g.
+// Jupyter.notebook.keyboard_manager.command_shortcuts.set_shortcut('alt-a', 'RISE:slideshow');
+// and likewise bind helper actions with e.g.
+// Jupyter.notebook.keyboard_manager.command_shortcuts.set_shortcut('shift-i', 'RISE:toggle-slide');
+    
 function setup() {
   $('head').append('<link rel="stylesheet" href=' + require.toUrl("./main.css") + ' id="maincss" />');
 
-  Jupyter.toolbar.add_buttons_group([
-    {
-    'label'   : 'Enter/Exit Live Reveal Slideshow',
-    'icon'    : 'fa-bar-chart-o',
-    'callback': function(){ revealMode(); },
-    'id'      : 'start_livereveal'
-    },
-  ]);
-  var document_keydown = function(event) {
-    if (event.which == 82 && event.altKey) {
-      revealMode();
-      return false;
-    }
-    return true;
-  };
-  $(document).keydown(document_keydown);
+  // use same label in button and shortcut
+  var rise_label = 'Enter/Exit RISE Slideshow';
+
+  // create button
+  Jupyter.toolbar.add_buttons_group([{
+    label   : rise_label,
+    icon    : 'fa-bar-chart-o',
+    callback: revealMode,
+    id      : 'RISE'
+  }]);
+  // define the slideshow action
+  var slideshow_action = {
+    help    : rise_label,
+    handler : revealMode,
```

Comment:
I am seeing this in my box

<img width="999" alt="screen shot 2017-10-31 at 6 45 20 pm" src="https://user-images.githubusercontent.com/1640669/32250778-9566cd36-be6c-11e7-90dc-fa9a0400efa6.png">

Do you see it as well?

Reply:
as I said I don't have this installed on my current machine, so I can't tell you for sure.

I 'm sure I have already seen something similar at some point; I guess this could be fixed by adding some data in slideshow_action, but I don't know how to this off the top of my head

Again if that's a big deal I would be willing to fix this in a subsequent PR

Reply:
> Again if that's a big deal I would be willing to fix this in a subsequent PR

That is OK for me.

Reply:
You were calling the action in the incorrect way, solved with this diff I am adding in the other branch:
```diff
diff --git a/rise/static/main.js b/rise/static/main.js
index f0eec6d..0c3fd49 100644
--- a/rise/static/main.js
+++ b/rise/static/main.js
@@ -802,7 +802,7 @@ function setup() {
   // register action
   Jupyter.notebook.keyboard_manager.actions.register(slideshow_action, "slideshow", "RISE");
   // bind action to keyboard shortcut
-  Jupyter.notebook.keyboard_manager.command_shortcuts.add_shortcut('alt-r', slideshow_action);
+  Jupyter.notebook.keyboard_manager.command_shortcuts.add_shortcut('alt-r', 'RISE:slideshow');

   // same with utility actions
   registerHelperActions();
```